src/components/TextInput/TextInputBase.tsx

```diff
@@ -46,6 +46,10 @@ const TextInputBase = ({
 
     const { View, StateLayer } = useMolecules();
     const { backgroundColor: parentBackground } = useContext(BackgroundContext);
+    const { inputStyle: _inputStyle } = useNormalizeStyles(
+        useMemo(() => ({ inputStyle }), [inputStyle]),
```

Comment:
inputStyle not necessary. if we are eventually going to destruct it anyway.
pass inputStyle directly.